TeachingRecordSystem/src/TeachingRecordSystem.Core/Services/Notify/HostApplicationBuilderExtensions.cs

```diff
@@ -22,6 +23,8 @@ public static IHostApplicationBuilder AddEmail(this IHostApplicationBuilder buil
             builder.Services.AddSingleton<INotificationSender, NoopNotificationSender>();
         }
 
+        builder.Services.AddTransient<SendEmailJob>();
```

Comment:
Do we definitely need this? We have emails going out already without this registration.